### PR TITLE
[Bugfix-22997] Add Big Sur to supported platforms

### PR DESCRIPTION
**Pull request by livecodealice**

### CODE REVIEW

File: docs/notes-base/platforms.md [hunk 1]
```diff
@@ -53,6 +53,7 @@ The Mac engine supports:
 * 10.13.x (High Sierra)
 * 10.14.x (Mojave)
 * 10.15.x (Catalina)
+* 11.x (Bug Sur)
```
Comment (livecodepanos) [inline docs/notes-base/platforms.md:57]: `Big Sur` :)

**Comment by livecodepanos** [comment]
@livecode-vulcan review ok d495a00

**Comment by livecode-vulcan** [comment]
💙 review by @livecodepanos ok d495a003e35917b2a06ddb76d488d1eff11f9a1a

**Comment by livecode-vulcan** [comment]
😎 test success d495a003e35917b2a06ddb76d488d1eff11f9a1a

* try-community-armv6-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-sdk26_ndk16r15/builds/931)
* try-community-armv7-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv7-android-ndk16r15/builds/339)
* try-community-arm64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-arm64-android-ndk16r15/builds/338)
* try-community-x86-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86-android-ndk16r15/builds/337)
* try-community-x86_64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86_64-android-ndk16r15/builds/338)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/2456)
* try-community-universal-ios-iphoneos14.1: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos14.1/builds/34)
* try-community-universal-ios-iphoneos13.5: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos13.5/builds/109)
* try-community-universal-ios-iphonesimulator14.1: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator14.1/builds/32)
* try-community-universal-ios-iphonesimulator13.5: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator13.5/builds/105)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/2483)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/2472)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/2470)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/2507)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/2443)

**Pull request merged**